examples/inference-deployments/mpt/README.md

```diff
@@ -7,6 +7,8 @@ Check out [this blog post](https://www.mosaicml.com/blog/mpt-7b) for more inform
 You’ll find in this folder:
 
 - Model YAMLS - read [docs](https://docs.mosaicml.com/projects/mcli/en/latest/inference/inference_schema.html) for an explanation of each field.
+    - `mpt_7b_simple.yaml` - an optimized no-code yaml to deploy [MPT-7B Base](https://huggingface.co/mosaicml/mpt-7b).
```

Comment:
Let's maybe not expose "no-code" because it's still a bunch of yaml inputs i.e. code. Perhaps say "semi automated" or "yam l with preset defaults for our mpt 7b model"?

Reply:
Talked with margaret and seems "no-code" will be the designator for these types of yamls so for consistency I think its fine that theyre called that